tests/common/widgets/hits/options.ts

```diff
@@ -126,6 +130,87 @@ export function createOptionsTests(
       `
       );
     });
+
+    skippableDescribe('instantsearch.js', skippedTests, () => {
```

Comment:
Right now we have two ways of writing these flavor-specific tests:
1. Have dedicated test files per flavor (e.g., `packages/instantsearch.js/src/widgets/hits/__tests__/hits.test.tsx`)
2. Use `skippableDescribe`

Should we pick one approach?

Reply:
I wasn't sure I would still need flavor specific widget tests files before, so I tried this with skippableDescribe, but forgot to get back to those later. I'll move the one for InstantSearch.js into its flavor test file and keep the skippable for the others as it's the more common one (to help remind us we'll need to harmonize the behavior in the future major of IS.js).